### DISCUSSION TITLE
Assertation failed after compile the modified engine

### DISCUSSION THREAD

**Discussion by cpsvt** [carla-simulator/carla#4412] [Q&A] [answered]
I have followed all the steps required to build the latest version of Carla using [this link](https://carla.readthedocs.io/en/latest/build_windows/) twice. 
Ever time I complete all the steps as structured and wanted to check if solution is compiled successfully (step4 in Unreal Engine section), I face the following error.  I'd appreciate it if someone could help what may cause this error( is it Unreal or Carla)?

[
![Capture_2](https://user-images.githubusercontent.com/86995899/126045783-fddee1fe-361c-4fea-8717-16bd40d2626b.PNG)
](url)

**Comment by corkyw10** [comment, marked as answer]
Can you confirm if the options explained in this section of the documentation are available to you:

> If the installation was successful, this should be recognised by Unreal Engine's version selector. You can check this by right-clicking on any .uproject file and selecting Switch Unreal Engine version. You should see a pop-up showing Source Build at PATH where PATH is the installation path that you have chosen. If you can not see this selector or the Generate Visual Studio project files when you right-click on .uproject files, something went wrong with the Unreal Engine installation and you will likely need to reinstall it correctly.

Also, I see you have specified 0.9.11 in your directory name. Be aware you are following documentation with a warning stating it is for the "dev" version of CARLA. If you intend to use 0.9.11 you will run into problems. If you want to install 0.9.11, change the documentation version in the bottom right corner of the screen, delete everything and start from scratch because the two versions use different versions of UE.
